### ISSUE TITLE
Carriage return included in json for /release endpoint

### ISSUE THREAD

**Issue by emir-hasanbegovic** [KineticCafe/cartage-rack#1]
The `/release` endpoint returns a return carriage in the `"release_hashref"` field as follows:

``` json
{
"env": "development",
"release_hashref": "(git) feature/deployment\n"
}
```

Expecting no return carriage as follows:

``` json
{
"env": "development",
"release_hashref": "(git) feature/deployment"
}
```

**Comment by halostatue** [comment]
This may already be fixed. Will verify.

**Comment by halostatue** [comment]
Already fixed in 5aee3542c3.
